docs/how-tos/deploy-with-helm.md

```diff
@@ -0,0 +1,43 @@
+# Deploy with Helm
+
+## Preface
+
+This guide will explain how to deploy Open Policy Agent (OPA) as part of your Helm-managed Kubernetes deployment.
+
+## Add the Chart Dependency
+
+To use the OPA instance in your deployment you should add the following to the `dependencies` section in your `Chart.yaml`:
+
+```yaml
+- name: opa
+  version: 0.1.0
+```
+
+You may additionally wish to add a condition, e.g. `opa.enabled`. If added you will need to set the corresponding value to true when you [Configure your Values](#configure-values)
+
+## Create Secret
+
+The default OPA configuration expects a secret named `bundler` containing `bundler-token`, this should be set to the Bearer Token required by the bundler service. The token can be obtained by reaching out via the [`#auth_auth` slack channel](https://diamondlightsource.slack.com/archives/C03P6QB9589). To create the secret in your namespace simply run:
```

Comment:
Should it be `beaer-token` rather than `bundler-token`?
https://github.com/DiamondLightSource/authz/blob/ae84943578133ab312d25c35614574c242c1dd1c/charts/opa/templates/deployment.yaml#L53

Reply:
It should indeed